posthog/settings/data_stores.py

```diff
@@ -117,10 +119,14 @@ def postgres_config(host: str) -> dict:
     DATABASES["replica"] = postgres_config(read_host)
     DATABASE_ROUTERS.append("posthog.dbrouter.ReplicaRouter")
 
+IS_CONNECTED_TO_PROD_PG_IN_DEBUG = DEBUG and (
```

Comment:
Nitpick: Check that `HOST` exists before and log a meaningful error if it doesn't?

Reply:
At this point `DATABASES["default"]["HOST"]` is guaranteed to be set! There's no situation in which there's no default (write) DB